web/impact/impact/tests/test_algolia_api_key_view.py

```diff
@@ -57,7 +55,7 @@ def _create_batch_program_and_named_group(status, batch_size):
 class TestAlgoliaApiKeyView(APITestCase):
     client_class = APIClient
     user_factory = UserFactory
-    url = reverse(AlgoliaApiKeyView.view_name)
+    url = "{}?index=mentor".format(reverse(AlgoliaApiKeyView.view_name))
```

Comment:
non blocking: using some library like url lib might be a little cleaner. might be overengineering. just calling it out

Reply:
doesn't look like django helps with things like these, recommendations I've seen do it like this as well. opted for a function that appends the relevant get parameter for the people and mentor directories